openoa/utils/met_data_processing.py

```diff
@@ -70,6 +70,27 @@ def circular_mean(x: pd.DataFrame | pd.Series | np.ndarray, axis: int = 0):
     )
 
 
+@series_method(data_cols=["u", "v"])
+def compute_wind_speed(
+    u: pd.Series | str, v: pd.Series | str, data: pd.DataFrame = None
+) -> pd.Series:
+    """Compute the wind speed from the u and v components. This function should only be used for
+    higher resolution wind data, otherwise the resulting wind speed will be lower than expected.
```

Comment:
I'd suggest a little explanation about the problem with using lower temporal resolution data to help users decide what resolution is acceptable. For example, vector-averaged u and v wind speed components will generally result in lower wind speed magnitudes than averages of the wind speed magnitude over the same time period. This becomes more severe as the averaging period increases.

Reply:
Added your snippet as a note and removed the abbreviated description.

Reply:
Looks good. I just added one more sentence. Feel free to change it though.